### PR TITLE
Amend and move the `Timing Songs With 8-Signatures` guide from the RC category

### LINKED ISSUE #6198: `Timing songs with #/8-signatures` should be in Guides instead of RC
it shouldn't mention RC at the beginning either, it doesn't set any rules and this article can stand on its own.
the first reference to it in RC is also misleading and can probably just be deleted

### PR DESCRIPTION
**Pull request by Gamingzone123**
closes #6198

### CODE REVIEW

File: wiki/Guides/Timing_Songs_With_8-Signatures/fr.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+outdated: true
+---
+
 # Timing des chansons avec des signatures en #/8
 
 De temps en temps, des musiques sont mappées avec une signature temporelle qui n'est pas divisée en quart de note (par exemple `3/4`, `4/4`, `7/4`, ...) mais en huitièmes de note (par exemple `6/8`, `7/8`, ...). Comme l'éditeur actuel ne prend pas en charge ces signatures #/8 et que nous ne savons pas si cela va changer dans un futur proche, un ajout a été fait à la règle concernant cette question sur la [section correspondante des critères de classement](/wiki/Ranking_Criteria#timing).
```
Comment (DoryanR) [inline wiki/Guides/Timing_Songs_With_8-Signatures/fr.md:7]:
```suggestion
De temps en temps, des musiques sont mappées avec une signature temporelle qui n'est pas divisée en quart de note (par exemple `3/4`, `4/4`, `7/4`, ...) mais en huitièmes de note (par exemple `6/8`, `7/8`, ...). L'éditeur actuel ne prend pas en charge ces signatures #/8 dans un futur proche. 
```

**Comment by TicClick** [comment]
1) remove references to the guide from the ranking criteria as per the second point of the issue you linked, and
2) add it to the `Guides` index page

**Comment by Gamingzone123** [comment]
> 1. remove references to the guide from the ranking criteria as per the second point of the issue you linked, and
> 2. add it to the `Guides` index page

done

File: wiki/Guides/Timing_Songs_With_8-Signatures/fr.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Timing des chansons avec des signatures en #/8
 
-De temps en temps, des musiques sont mappées avec une signature temporelle qui n'est pas divisée en quart de note (par exemple `3/4`, `4/4`, `7/4`, ...) mais en huitièmes de note (par exemple `6/8`, `7/8`, ...). Comme l'éditeur actuel ne prend pas en charge ces signatures #/8 et que nous ne savons pas si cela va changer dans un futur proche, un ajout a été fait à la règle concernant cette question sur la [section correspondante des critères de classement](/wiki/Ranking_Criteria#timing).
+De temps en temps, des musiques sont mappées avec une signature temporelle qui n'est pas divisée en quart de note (par exemple `3/4`, `4/4`, `7/4`, ...) mais en huitièmes de note (par exemple `6/8`, `7/8`, ...). L'éditeur actuel ne prend pas en charge ces signatures #/8 dans un futur proche.
```
Comment (Walavouchey) [inline wiki/Guides/Timing_Songs_With_8-Signatures/fr.md:3]:
```suggestion
De temps en temps, des musiques sont mappées avec une signature temporelle qui n'est pas divisée en quart de note (par exemple `3/4`, `4/4`, `7/4`, ...) mais en huitièmes de note (par exemple `6/8`, `7/8`, ...). L'éditeur actuel ne prend pas en charge ces signatures #/8 dans un futur proche.
```

**Review by Walavouchey — COMMENTED** [review]
I've fixed ci errors and added a redirect, even though there are no other references to this article apart from rc

**Review by Walavouchey — DISMISSED** [review]

**Comment by TicClick** [comment]
the RC translations need a single edit each, hold on

File: wiki/Ranking_Criteria/ru.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 outdated: true
+outdated_since: 7742320aaa249169bcb0536d8fc19dc7d32015fe
```
Comment (TicClick) [inline wiki/Ranking_Criteria/ru.md:3]: uhh this is not true (it's already outdated since we don't know when, so you can't add `outdated_since`)

**Review by TicClick — APPROVED** [review]

File: wiki/Ranking_Criteria/ru.md (hunk 3 above)
Comment (Gamingzone123) [reply wiki/Ranking_Criteria/ru.md]: that particular one wasn't already marked as outdated, so I assumed it was up to date prior to my change

**Pull request merged**